annotations/annotations.go

```diff
@@ -25,4 +25,7 @@ const (
 
 	// OpenShiftLongDescriptionAnnotation is a resource's long description
 	OpenShiftLongDescriptionAnnotation = "openshift.io/long-description"
+
+	// OpenShiftComponent is a common, optional annotation that stores the owning component for a resource.
```

Comment:
When you say owning component, to me, that is open to interpretation as to what the expected value would be.
Can you expand this to add a bit more context/an example of the expected value? 

Is it the Jira component? OCPBUGS component? ClusterOperator name? Repository?

Reply:
updated docs and added example.